modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/FDSEntries.tsx

```diff
@@ -794,13 +799,27 @@ const FDSEntries = ({
 		});
 	};
 
+	const NameRenderer = ({itemData}: {itemData: FDSEntryType}) => {
+		return (
+			<div className="table-list-title">
+				<ClayLink href={getViewURL(itemData).toString()}>
+					{itemData.label}
+				</ClayLink>
+			</div>
+		);
+	};
+
 	const views = [
 		{
 			contentRenderer: 'table',
 			name: 'table',
 			schema: {
 				fields: [
-					{fieldName: 'label', label: Liferay.Language.get('name')},
+					{
+						contentRenderer: 'name',
+						fieldName: 'label',
+						label: Liferay.Language.get('name'),
+					},
```

Comment:
Using `contentRenderer: 'actionLink'` should work here. This is the exact case for it, a string cell content that duplicates an action. If no `actionId` is set, it will take the first item in actions list. This will work, as `view` is the first action, but we may want to future-proof it by setting something like `field.actionId = 'view'` and `action.data.id = 'view'`.

But! I checked this and it still doesn't work. That is because we have very poor support for `onClick` in action link renderer. Instead of [this monstrosity](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/cell_renderers/ActionLinkRenderer.js#L96-L102), it should be something like:
```js
else if (currentAction.onClick) {
	currentAction.onClick({itemData});
}
```